themes/cardinal_service/pdb_components/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "pdb_components",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production"
```

Comment:
I think we have been using the conventions of `npm/yarn build` for development builds and `npm/yarn publish` for production builds.